fastembed/rerank/cross_encoder/onnx_text_model.py

```diff
@@ -0,0 +1,77 @@
+from typing import Sequence, Optional
+from pathlib import Path
+
+from onnxruntime import InferenceSession
+from tokenizers import Tokenizer
+from tokenizers.processors import TemplateProcessing
+
+from fastembed.common.onnx_model import OnnxModel, OnnxProvider
+from fastembed.common.preprocessor_utils import load_tokenizer
+
+
+class OnnxCrossEncoderModel(OnnxModel):
+    def __init__(
+        self,
+        model_name: str,
+        cache_dir: str = None,
+        threads: int = None,
+        providers: Sequence[OnnxProvider] = None,
+        **kwargs,
+    ):
+        self.tokenizer: Tokenizer = None
+        self.model: InferenceSession = None
+        self.model_name: str = model_name
+        self.cache_dir: str = cache_dir
+        self.threads: int = threads
+        self.providers: Sequence[OnnxProvider] = providers
```

Comment:
we don't need to have init here, we use it only for variable declaration, tokenizer is already declared in the parent class